api_types.ts

```diff
@@ -515,6 +515,20 @@ export interface FetchRequest {
    * wish to make an unauthenticated supporting request as part of a formula implementation.
    */
   disableAuthentication?: boolean;
+  /**
+   * By default, any fetch response that more than 4MB will be rejected with
```

Comment:
Nit, would change to something like "any fetch response that is greater than a Coda-enforced limit (currently 4MB) will be rejected with ..."